components/core/src/error.rs

```diff
@@ -117,6 +125,17 @@ impl fmt::Display for Error {
                          origin/name (example: acme/redis)",
                         e)
             }
+            Error::InvalidPackageTarget(ref e) => {
+                format!("Invalid package target: {:?}. A valid target is in the form \
```

Comment:
Just `{}` is fine here, `{:?}` is the "debug" output and not the "display" output of your type